app/views/dashboards/_tags.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= javascript_packs_with_chunks_tag "dashboardTagsDisableUnchangedButtons", defer: true %>
```

Comment:
`following_tags` and `hidden_tags` have some overlap in its templates hence I've pulled it out into a partial.

Reply:
I've also removed some HTML and styling in favour of moving towards the new design attached to the issue.